pkg/proxy/options/authentication.go

```diff
@@ -55,7 +56,7 @@ func NewAuthentication() *Authentication {
 			WithServiceAccounts().
 			WithTokenFile(),
 		// when adding new auth methods, also update AdditionalAuthEnabled below
-		DropGroups: []string{user.SystemPrivilegedGroup},
+		DropGroups: []string{user.SystemPrivilegedGroup, bootstrap.SystemLogicalClusterAdmin},
```

Comment:
can we add a comment why we are doing this for `SystemLogicalClusterAdmin` ?

Reply:
Done